testkit/tests/counter/main.rs

```diff
@@ -545,11 +548,12 @@ fn test_explorer_transaction() {
         let (pubkey, key) = crypto::gen_keypair();
         TxIncrement::new(&pubkey, 5, &key)
     };
-    let info: Value = api.get_err(
+    let info: ApiError = api.get_err(
         ApiKind::Explorer,
         &format!("v1/transactions/{}", &tx.hash().to_string()),
     );
-    assert_eq!(info, json!({ "type": "unknown" }));
+    let error_body = json!({ "type": "unknown" }).to_string();
+    assert_matches!(info, ApiError::NotFound(ref body) if *body == error_body);
```

Comment:
This code here imo signifies that the chosen approach is not perfect. The comparison `*body == error_body` is logically incorrect; a correct one would be

```rust
ApiError::NotFound(ref body) if serde_json::to_value(body).unwrap() ==
    json!({ "type": "unknown" })
```

Maybe, make `NotFound` variant in `ApiError` wrap `serde_json::Value`?